samples/v2/precompiled/HttpStart.cs

```diff
@@ -16,13 +16,14 @@ public static class HttpStart
         [FunctionName("HttpStart")]
         public static async Task<HttpResponseMessage> Run(
             [HttpTrigger(AuthorizationLevel.Function, methods: "post", Route = "orchestrators/{functionName}")] HttpRequestMessage req,
-            [OrchestrationClient] DurableOrchestrationClientBase starter,
+            [OrchestrationClient] IDurableOrchestrationClient starter,
             string functionName,
             ILogger log)
         {
             // Function input comes from the request content.
             dynamic eventData = await req.Content.ReadAsAsync<object>();
-            string instanceId = await starter.StartNewAsync(functionName, eventData);
+            string instanceId = Guid.NewGuid().ToString();
+            await starter.StartNewAsync(functionName, instanceId, eventData);
```

Comment:
What was the reason for changing this logic?  Does `StartNewAsync` no longer return the instance ID?